internal/assets/manifests/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -54,6 +54,8 @@ metadata:
     istio.io/rev: {{ include "namespaced-revision" . }}
     app: sidecar-injector
     release: {{ .Release.Name }}
+  annotations:
+    admissions.enforcer/disabled: "true"
```

Comment:
adding link to docs for reference: https://learn.microsoft.com/en-us/azure/aks/faq#can-admission-controller-webhooks-impact-kube-system-and-internal-aks-namespaces